app/styles/authors.scss

```diff
@@ -0,0 +1,61 @@
+.vert-align .form-group {
+    margin-top: 15px;
+}
+
+.bib-padding {
+    text-align: center;
+}
+
+// Author table divisions
+.author-table {
+    td {
+        border-top: 1px solid #ddd;
+    }
+}
+
+.col-division-left {
+    padding-left: 20px;
+    border-left: 1px solid #ccc;
+}
+
+.col-division-right {
+    border-right: 1px solid #ccc;
+}
+
+.search-box {
```

Comment:
That's one way to do it. Though that's not so much namespacing, as making sure that your CSS selector only applies in certain circumstances, based on the markup of other elements around where this is used. 

There's a few approaches to organizing CSS and naming things, by some pretty clever people- you wouldn't think it's a subject for study, but it really is! So several valid approaches, but calling it `search-box` as a one-off class is just a smidge too far on the generic end.